packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * When true, will disable the field.
+  */
+  isDisabled: PropTypes.bool,
+
+  /**
+   * Minimum value in the range.
+  */
+  minimumValue: PropTypes.number.isRequired,
+
+  /**
+   * Maximum value in the range.
+  */
+  maximumValue: PropTypes.number.isRequired,
+
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
```

Comment:
I thought the minimum and maximum values would be used for the labels based on the callouts?